src/initial-submission/components/DetailsForm.test.tsx

```diff
@@ -357,4 +359,108 @@ describe('DetailsForm', (): void => {
             expect(getByLabelText('details.second-cosubmission-title-label')).toBeInTheDocument();
         });
     });
+
+    describe('validation', () => {
+        const renderDetails = (): RenderResult => {
+            return render(
+                <DetailsForm
+                    initialValues={{
+                        id: 'blah',
+                        updated: Date.now(),
+                        articleType: '',
+                    }}
+                    ButtonComponent={({
+                        triggerValidation,
+                    }: {
+                        _: Function;
+                        triggerValidation: () => Promise<boolean>;
+                    }): JSX.Element => (
+                        <button
+                            onClick={(): void => {
+                                triggerValidation();
+                            }}
+                        >
+                            TEST BUTTON
+                        </button>
+                    )}
+                />,
+            );
+        };
+
+        it('shows error message if title is empty', async () => {
+            const { getByText } = renderDetails();
+
+            fireEvent.click(getByText('TEST BUTTON'));
+            await waitFor(() => expect(getByText('details.validation.title-required')).toBeInTheDocument());
+        });
+
+        it('shows error message if subjects is empty', async () => {
```

Comment:
We should probably add a test case for the `feature` article type not requiring a min set of values

Reply:
How would that work with "subject-min" ... does this mean the message needs to be dynamic?

Reply:
Oh! Just read the code below... yes probably a good idea to test that branch in validation :)

Reply:
Add the feature test case 👍